Project/Libs/DebugDraw/debug_draw.hpp

```diff
@@ -173,9 +173,9 @@
 // yourself before including this file.
 //
 #ifndef DEBUG_DRAW_CXX11_SUPPORTED
-    #if (__cplusplus > 199711L)
-        #define DEBUG_DRAW_CXX11_SUPPORTED 1
-    #endif // __cplusplus
+#if (__cplusplus > 199711L)
+#define DEBUG_DRAW_CXX11_SUPPORTED 1
+#endif // __cplusplus
```

Comment:
Was there any edit in debug_draw? If not, the formatting should be reverted.

Reply:
Yes, there was. On the sphere function

Reply:
Is there any way to revert the formatting so that the changes are more apparent?